src/lib/babel-custom.js

```diff
@@ -46,7 +46,8 @@ const createConfigItems = (babel, type, items) => {
 	});
 };
 
-const presetEnvRegex = RegExp(/@babel\/(preset-)?env/);
+const environmentPreset = '@babel/preset-env';
```

Comment:
This no longer finds `@babel/env`, which is an alias of `@babel/preset-env`

Reply:
Oh I searched for it and couldn't find any usage of it. i'll create a follow up PR

Reply:
Heh - yeah we don't have a test for it, just it's possible to get that value when we read in user config.